src/client/renderingengine.h

```diff
@@ -138,6 +138,16 @@ class RenderingEngine
 			const irr::core::dimension2d<u32> initial_screen_size,
 			const bool initial_window_maximized);
 
+	static bool shouldRender()
+	{
+#ifdef __ANDROID__
+		// On Android, pause rendering while the app is paused.
```

Comment:
```suggestion
	{
		// On Android, pause rendering while the app is in background (generally not visible).
		// Don't do this on desktop because windows can be partially visible.
#ifdef __ANDROID__
```